CHANGELOG.md

```diff
@@ -248,6 +248,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Empty lines in selections are now properly copied to the clipboard
 - Selection start point lagging behind initial cursor position
 - Rendering of selections which start above the visible area and end below it
+- Mouse press no longer ignored in padding width
```

Comment:
I think we could be a little more precise here. Since the mouse press is still ignored in regards to the PTY (as you've previously pointed out).

So I'd recommend explicitly stating that it is now possible to start selection from within the padding.

Reply:
Made the comment more explicit. The change includes allowing following links in the first or last column by clicking in the padding (or other mouse interactions excluding pty propagation).